dockerfile_parse/parser.py

```diff
@@ -149,9 +151,16 @@ def _open_dockerfile(self, mode):
             yield self.fileobj
             self.fileobj.seek(0)
         else:
-            with open(self.dockerfile_path, mode) as dockerfile:
+            with self.dockerfile.open(mode) as dockerfile:
                 yield dockerfile
 
+    @property
+    def dockerfile_path(self):
+        """
+        :return: Path to the Dockerfile as a string, or None if using fileobj
+        """
+        return str(self.dockerfile) if self.dockerfile else None
```

Comment:
I think it was previously possible to set `dockerfile_path` after the object was already created. It wasn't necessarily an _intended_ use case, but to avoid breaking changes, could `dockerfile_path` also have a setter?